pre-commit

```diff
@@ -1,9 +1,10 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
`[[` on L34 is not a POSIX shell feature according to shellcheck.

Reply:
True story :)